entgql/template/node.tmpl

```diff
@@ -65,14 +65,14 @@ type Edge struct {
 {{/* loop over all types and add implement the Node interface. */}}
 {{ range $n := $gqlNodes }}
 	{{ $receiver := $n.Receiver }}
-		{{- $marshalID := marshalerID $n.ID.Type $.Annotations.EntGQLConfig }}
-		{{- if $marshalID }}
+	{{ $marshalID := gqlMarshaler $n.ID }}
+	{{- if gqlMarshaler $n.ID }}
 		func ({{ $receiver }} {{ $n.Name }}) marshalID() string {
-			var buf bytes.Buffer
-			{{ $receiver }}.ID.MarshalGQL(&buf)
-			return buf.String()
+			var b bytes.Buffer
```

Comment:
Be careful, it maybe conflict with the receiver.

Reply:
Right! Thanks 🙏🏻